src/libs/conduit/conduit_node.cpp

```diff
@@ -9513,6 +9513,147 @@ Node::to_double_array(Node &res) const
 
 }
 
+//---------------------------------------------------------------------------//
+void
+Node::to_data_type(index_t dtype_id, Node &res) const
+{
+    switch(dtype_id)
+    {
+        /* ints */
+        case DataType::INT8_ID:
+        {
+            if(this->dtype().number_of_elements() == 1)
+            {
```

Comment:
do we need the 1 ele cases? I think to_array will handle both?

Reply:
I just tested this out and you are correct! I'll amend this with my next set of changes.